packages/logging/src/shared.ts

```diff
@@ -0,0 +1,13 @@
+export const Version = __PACKAGE_VERSION__;
```

Comment:
I would rename this file to `runtime.ts` as it detects which runtime is the library being run in.